addons/common/functions/fnc_checkFiles.sqf

```diff
@@ -52,23 +52,25 @@ _addons = _addons select {_x find "ace_" == 0};
 ///////////////
 // check dlls
 ///////////////
-{
-    private _versionEx = _x callExtension "version";
+if (productVersion select 6 == "Windows") then {
```

Comment:
https://community.bistudio.com/wiki/productVersion
>String (Arma 3) - Platform ("Windows", "Linux" or "OSX")

how else would you do that?

>It would be Better to inverse it and check for the equivalent for Linux/Mac.

`.so` and the MacOS equivalent are not supported by our code and would require further changes in other places too. What do you mean by "inverse"?